docs/app/services/playground/strategies/playground-strategy.ts

```diff
@@ -31,7 +34,28 @@ export abstract class PlaygroundStrategy {
 
     /** Get all the external styles that should be loaded in the `head` of the HTML */
     getGlobalExternalStyles(assetsUrl: string): string[] {
-        return [];
+        const stylesheets = [
```

Comment:
I don't think this should be added to the base abstract class. I think you can extend the strategy pattern as described previously to keep it clean. Let me know what was unclear and I can try to explain further.